Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -3432,6 +3432,7 @@ def handle_eml(file_path, b64=False, file_name=None, parse_only_headers=False, m
             # decode bytes taking into account BOM and re-encode to utf-8
             file_data = file_data.decode("utf-8-sig").encode("utf-8")
 
+        file_data = file_data.replace('\r\n..', '\r\n.')
```

Comment:
will there always be `\r`?

Reply:
On the few examples I found `\r` apeared